android/sdl_android/src/main/java/com/smartdevicelink/util/SdlAppInfo.java

```diff
@@ -49,16 +59,18 @@
     //FIXME we shouldn't be duplicating constants, but this currently keeps us from needing a context instance.
     private static final String SDL_ROUTER_VERSION_METADATA = "sdl_router_version";
     private static final String SDL_CUSTOM_ROUTER_METADATA = "sdl_custom_router";
+    private static final String SDL_OEM_VEHICLE_TYPE_METADATA = "sdl_oem_vehicle_type";
 
 
     String packageName;
     ComponentName routerServiceComponentName;
     int routerServiceVersion = 4; //We use this as a default and assume if the number doesn't exist in meta data it is because the app hasn't updated.
     boolean isCustomRouterService = false;
+    List<VehicleType> vehicleMakesList = new ArrayList<>();
     long lastUpdateTime;
 
 
-    public SdlAppInfo(ResolveInfo resolveInfo, PackageInfo packageInfo) {
+    public SdlAppInfo(ResolveInfo resolveInfo, PackageInfo packageInfo, Context context) {
```

Comment:
@kostyaBoss `SdlAppInfoTests.java`  is not building because it is referring to the 2 argument constructor.  Please correct this file so that it builds.  Bringing back the 2 argument constructor may help resolve it.